mesonbuild/modules/keyval.py

```diff
@@ -55,7 +55,7 @@ def _load_file(path_to_config: str) -> T.Dict[str, str]:
         return result
 
     @noKwargs
-    @typed_pos_args('keyval.laod', (str, mesonlib.File))
+    @typed_pos_args('keyval.load', (str, mesonlib.File))
```

Comment:
Scary

Reply:
Well, it's just the text for the error message if you use invalid arguments to the load() function. Pretty silly but non-impactful other than as a source of confusion.